frontend/.prettierrc

```diff
@@ -1,10 +1,13 @@
 {
   "plugins": ["prettier-plugin-tailwindcss"],
   "tailwindFunctions": ["clsx", "cn", "cva"],
-  "printWidth": 90,
+  "printWidth": 100,
+  "tabWidth": 2,
   "singleQuote": false,
-  "quoteProps": "as-needed",
-  "trailingComma": "es5",
-  "bracketSpacing": true,
-  "bracketSameLine": true
```

Comment:
You can keep your previous settings if you feel comfortable with them, the ones that I suggested is what I usually use